### PR TITLE
Switched "ingest" -> "visualize" in reference to the Promethius metrics

### PR DESCRIPTION
**Pull request by dtaivpp**
Switched the wording from ingest to visualize to better represent what is possible with Prometheus and OpenSearch Dashboards. 

Signed-off-by: David Tippett <dtip@amazon.com>

### Description
Updated word for technical accuracy. 
 
### Issues Resolved
N/A

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by dtaivpp** [comment]
Ran this by @anirudha  and Nitin to verify that we do not actually ingest the data but just allow users to visualize and they confirmed it.

**Comment by krisfreedain** [comment]
thanks @dtaivpp - @jhmcintyre can you take a look at this - with your input we'll push out this update quickly.

**Comment by jhmcintyre** [comment]
LGTM

**Review by jhmcintyre — APPROVED** [review]

**Review by krisfreedain — CHANGES_REQUESTED** [review]
small spelling fix

File: _posts/2022-11-15-opensearch-2-4-is-available-today.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To enhance security administration, we’ve also added a feature that [lets you
...
+With this release, you can now visulize metrics data from the open-source [Prometheus](https://prometheus.io/) monitoring solution as well as from log data aggregated within OpenSearch, unlocking new observability capabilities that allow you to analyze and correlate data across logs and traces as well as metrics. Previously, the OpenSearch observability plugin only allowed you to ingest and visualize logs and traces from your monitored environments. With [this feature](https://github.com/opensearch-project/sql/issues/561), you can observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform root cause analysis.
```
[the remark this answers is not in the document]
Comment (dtaivpp) [reply _posts/2022-11-15-opensearch-2-4-is-available-today.md:43]: oof

**Review by krisfreedain — APPROVED** [review]
thanks @dtaivpp

**Pull request merged**